crates/bitwarden/src/crypto/enc_string/asymmetric.rs

```diff
@@ -178,6 +177,44 @@ impl AsymmEncString {
     }
 }
 
+impl KeyEncryptable<AsymmetricCryptoKey, AsymmEncString> for &[u8] {
+    fn encrypt_with_key(self, _key: &AsymmetricCryptoKey) -> Result<AsymmEncString> {
+        todo!("implement encrypt")
+    }
+}
```

Comment:
Any benefit to implementing this vs excluding it? It should cause a compile error without it which is preferable over runtime.

Reply:
That's a good point yeah, removed